libraries/codegen-tooling/inc/can_msg_defs.h

```diff
@@ -35,7 +35,7 @@ typedef enum {
   SYSTEM_CAN_MESSAGE_OVUV_DCDC_AUX = 16,
   SYSTEM_CAN_MESSAGE_MC_ERROR_LIMITS = 17,
   SYSTEM_CAN_MESSAGE_MOTOR_CONTROLS = 18,
-  SYSTEM_CAN_MESSAGE_LIGHTS_STATES = 24,
+  SYSTEM_CAN_MESSAGE_LIGHTS_GENERIC_TYPES = 24,
```

Comment:
What is this supposed to represent?

Reply:
There's `generic lights` and there's `signal lights`, generic lights are all the lights that don't require blinking. Since they all get directly passed to `lights_gpio`, I thought it'd be easier to have the messages set like this so I don't have gigantic look-up tables.

Reply:
Changed this to `lights types` and `lights signal types`

Reply:
nvm, `lights` and `lights signal`

Reply:
Update: `light_state` and `signal_light_state`